2021-12-12 10:49:53,526 INFO    MainThread:3634 [wandb_setup.py:_flush():69] setting env: {'run_group': 'development', 'project': 'nyc_airbnb'}
2021-12-12 10:49:53,527 WARNING MainThread:3634 [wandb_setup.py:_flush():69] could not save program above cwd: /opt/conda/envs/mlflow-d1fd4b13aac386abe6aa5a2ada1ca11214902ce4/bin/pytest
2021-12-12 10:49:53,527 INFO    MainThread:3634 [wandb_setup.py:_flush():69] setting login settings: {}
2021-12-12 10:49:53,528 INFO    MainThread:3634 [wandb_init.py:_log_setup():334] Logging user logs to /home/src/data_check/wandb/run-20211212_104953-fq3w6coy/logs/debug.log
2021-12-12 10:49:53,529 INFO    MainThread:3634 [wandb_init.py:_log_setup():335] Logging internal logs to /home/src/data_check/wandb/run-20211212_104953-fq3w6coy/logs/debug-internal.log
2021-12-12 10:49:53,531 INFO    MainThread:3634 [wandb_init.py:init():367] calling init triggers
2021-12-12 10:49:53,532 INFO    MainThread:3634 [wandb_init.py:init():372] wandb.init called with sweep_config: {}
config: {}
2021-12-12 10:49:53,533 INFO    MainThread:3634 [wandb_init.py:init():416] starting backend
2021-12-12 10:49:53,533 INFO    MainThread:3634 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-12-12 10:49:53,548 INFO    MainThread:3634 [backend.py:ensure_launched():123] starting backend process...
2021-12-12 10:49:53,551 INFO    MainThread:3634 [backend.py:ensure_launched():127] started backend process with pid: 3651
2021-12-12 10:49:53,553 INFO    MainThread:3634 [wandb_init.py:init():421] backend started and connected
2021-12-12 10:49:53,555 INFO    MainThread:3634 [wandb_init.py:init():461] updated telemetry
2021-12-12 10:49:53,556 INFO    MainThread:3634 [wandb_init.py:init():480] communicating current version
2021-12-12 10:49:54,166 INFO    MainThread:3634 [wandb_init.py:init():485] got version response upgrade_message: "wandb version 0.12.7 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-12-12 10:49:54,168 INFO    MainThread:3634 [wandb_init.py:init():493] communicating run to backend with 30 second timeout
2021-12-12 10:49:55,200 INFO    MainThread:3634 [wandb_init.py:init():518] starting run threads in backend
2021-12-12 10:49:55,249 INFO    MainThread:3634 [wandb_run.py:_console_start():1476] atexit reg
2021-12-12 10:49:55,253 INFO    MainThread:3634 [wandb_run.py:_redirect():1339] redirect: SettingsConsole.REDIRECT
2021-12-12 10:49:55,255 INFO    MainThread:3634 [wandb_run.py:_redirect():1344] Redirecting console.
2021-12-12 10:49:55,256 INFO    MainThread:3634 [redirect.py:install():213] install start
2021-12-12 10:49:55,257 INFO    MainThread:3634 [redirect.py:install():228] install stop
2021-12-12 10:49:55,257 INFO    MainThread:3634 [redirect.py:install():213] install start
2021-12-12 10:49:55,258 INFO    MainThread:3634 [redirect.py:install():228] install stop
2021-12-12 10:49:55,259 INFO    MainThread:3634 [wandb_run.py:_redirect():1390] Redirects installed.
2021-12-12 10:49:55,259 INFO    MainThread:3634 [wandb_init.py:init():542] run started, returning control to user process
2021-12-12 10:50:01,537 INFO    MainThread:3634 [wandb_setup.py:_flush():69] setting env: {'run_group': 'development', 'project': 'nyc_airbnb'}
2021-12-12 10:50:01,538 WARNING MainThread:3634 [wandb_setup.py:_flush():69] could not save program above cwd: /opt/conda/envs/mlflow-d1fd4b13aac386abe6aa5a2ada1ca11214902ce4/bin/pytest
2021-12-12 10:50:01,539 INFO    MainThread:3634 [wandb_setup.py:_flush():69] setting login settings: {}
2021-12-12 10:50:01,540 INFO    MainThread:3634 [wandb_init.py:_log_setup():334] Logging user logs to /home/src/data_check/wandb/run-20211212_105001-fq3w6coy/logs/debug.log
2021-12-12 10:50:01,541 INFO    MainThread:3634 [wandb_init.py:_log_setup():335] Logging internal logs to /home/src/data_check/wandb/run-20211212_105001-fq3w6coy/logs/debug-internal.log
2021-12-12 10:50:01,542 INFO    MainThread:3634 [wandb_init.py:init():367] calling init triggers
2021-12-12 10:50:01,543 INFO    MainThread:3634 [wandb_init.py:init():372] wandb.init called with sweep_config: {}
config: {}
2021-12-12 10:50:01,544 INFO    MainThread:3634 [wandb_init.py:init():413] wandb.init() called when a run is still active
2021-12-12 10:50:04,197 INFO    MainThread:3634 [wandb_run.py:_atexit_cleanup():1446] got exitcode: 0
2021-12-12 10:50:04,201 INFO    MainThread:3634 [wandb_run.py:_restore():1418] restore
2021-12-12 10:50:04,202 INFO    MainThread:3634 [redirect.py:uninstall():232] uninstall start
2021-12-12 10:50:04,203 INFO    MainThread:3634 [redirect.py:_stop():287] _stop: stdout
2021-12-12 10:50:04,204 INFO    MainThread:3634 [redirect.py:_stop():293] _stop closed: stdout
2021-12-12 10:50:04,204 INFO    stdout    :3634 [redirect.py:_pipe_relay():129] relay done saw last write: stdout
2021-12-12 10:50:04,206 INFO    stdout    :3634 [redirect.py:_pipe_relay():145] relay done done: stdout
2021-12-12 10:50:04,206 INFO    MainThread:3634 [redirect.py:_stop():299] _stop joined: stdout
2021-12-12 10:50:04,207 INFO    MainThread:3634 [redirect.py:_stop():301] _stop rd closed: stdout
2021-12-12 10:50:04,208 INFO    MainThread:3634 [redirect.py:uninstall():236] uninstall done
2021-12-12 10:50:04,209 INFO    MainThread:3634 [redirect.py:uninstall():232] uninstall start
2021-12-12 10:50:04,210 INFO    MainThread:3634 [redirect.py:_stop():287] _stop: stderr
2021-12-12 10:50:04,211 INFO    MainThread:3634 [redirect.py:_stop():293] _stop closed: stderr
2021-12-12 10:50:04,211 INFO    stderr    :3634 [redirect.py:_pipe_relay():129] relay done saw last write: stderr
2021-12-12 10:50:04,213 INFO    stderr    :3634 [redirect.py:_pipe_relay():145] relay done done: stderr
2021-12-12 10:50:04,214 INFO    MainThread:3634 [redirect.py:_stop():299] _stop joined: stderr
2021-12-12 10:50:04,214 INFO    MainThread:3634 [redirect.py:_stop():301] _stop rd closed: stderr
2021-12-12 10:50:04,215 INFO    MainThread:3634 [redirect.py:uninstall():236] uninstall done
2021-12-12 10:50:04,630 INFO    MainThread:3634 [wandb_run.py:_wait_for_finish():1568] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 726
  total_bytes: 726
}

2021-12-12 10:50:06,637 INFO    MainThread:3634 [wandb_run.py:_wait_for_finish():1568] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 726
  total_bytes: 1567
}

2021-12-12 10:50:08,647 INFO    MainThread:3634 [wandb_run.py:_wait_for_finish():1568] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1567
  total_bytes: 1567
}

2021-12-12 10:50:09,733 INFO    MainThread:3634 [wandb_run.py:_show_files():1790] logging synced files
